staramr/tests/unit/results/test_AMRDetectionSummary.py

```diff
@@ -400,3 +410,229 @@ def testPointfinderResfinderSingleGene(self):
 
         self.assertEqual('file1', summary.index[0], 'File name not equal')
         self.assertEqual('blaIMP-42, gyrA', summary['Genotype'].iloc[0], 'Genes not equal')
+
+    def testDetailedSummary_noPlasmid_noPoint(self):
```

Comment:
Amazing set of tests. Thanks so much for adding these :smile:

Reply:
No problem!